### PR TITLE
Update pyproject.toml

### PR DESCRIPTION
**Pull request by asherf**
as far as I can tell this file is not coming from our monorepo.
I am assuming the email I used is the correct one, @wvalori should I use a different one?
We probably want to use a different license than MIT, no ?
@stgarrity @james-uc 
Screenshot 2023-09-21 at 11 05 39 AM

### CODE REVIEW

**Review by james-uc — COMMENTED** [review]
Other than the missing quotes, looks good

File: pyproject.toml [hunk 1]
```diff
@@ -8,7 +8,15 @@ build-backend = "setuptools.build_meta"
 name = "usercloudssdk"
 version = "1.0.10"
 description = "Python SDK for UserClouds"
-authors = [{ name = "Real Python", email = "info@realpython.com" }]
+authors = [{ name = "UserClouds", email = "info@userclouds.com" }]
+classifiers = [
+    Development Status :: 4 - Beta
```
[the remark this answers is not in the document]
Comment (asherf) [reply pyproject.toml:13]: 🤦🏼

**Comment by wvalori** [comment]
yeah I think info@userclouds.com is good. vlad, steve and I are on that. I don't know how the MIT license got in there - might be an @stgarrity question?

**Comment by stgarrity** [comment]
> yeah I think [info@userclouds.com](mailto:info@userclouds.com) is good. vlad, steve and I are on that. I don't know how the MIT license got in there - might be an @stgarrity question?

I think we just chose a simple, non-restrictive one since the SDK code isn't really something we need to protect? Eg. if people want to mimic our APIs that's a good thing, and any friction that prevents someone from using it (even just a license they don't recognize) is a bad thing? Absolutely open to changing it, but I believe those are the right principles?

**Comment by wvalori** [comment]
great. makes sense - I don't know anything about licenses

**Pull request merged**